### PR TITLE
Enable stubtest for six

### PR DESCRIPTION
**Pull request by srittau**

### CODE REVIEW

**Comment by Akuli** [comment]
This seems like we would have to duplicate much of stdlib allowlists, and the allowlists in `six` would get outdated easily.

**Comment by srittau** [comment]
> This seems like we would have to duplicate much of stdlib allowlists, and the allowlists in `six` would get outdated easily.

I'm going through the allowlist now, checking what's from the stdlib and what's genuine.

**Comment by srittau** [comment]
@Akuli I've now ignored stubtest errors in `six.moves`, because as you said, they are just stdlib errors. But outside `moves`, stubtest spotted a few genuine problems.

File: stubs/six/six/__init__.pyi [hunk 1]
```diff
@@ -1,10 +1,12 @@
 from __future__ import print_function
 
+import builtins
 import types
 import unittest
 from builtins import next as next
 from collections.abc import Callable, ItemsView, Iterable, Iterator as _Iterator, KeysView, Mapping, ValuesView
 from functools import wraps as wraps
+from importlib.util import spec_from_loader as spec_from_loader
```
Comment (Akuli) [inline stubs/six/six/__init__.pyi:9]: This won't work on Python 2.

File: stubs/six/six/__init__.pyi [hunk 2]
```diff
@@ -30,9 +32,12 @@ binary_type = bytes
 
 MAXSIZE: int
 
-def callable(obj: object) -> bool: ...
+callable = builtins.callable
+
 def get_unbound_function(unbound: types.FunctionType) -> types.FunctionType: ...
-def create_bound_method(func: types.FunctionType, obj: object) -> types.MethodType: ...
+
+create_bound_method = types.MethodType
```
Comment (Akuli) [inline stubs/six/six/__init__.pyi:39]: This won't work on Python 2 either.

File: stubs/six/six/__init__.pyi (hunk 1 above)
Comment (srittau) [reply stubs/six/six/__init__.pyi:9]: These stubs are Python 3 only. Python 2 stubs are in the `@python2` directory.

**Review by Akuli — APPROVED** [review]

**Pull request merged**